automation/test.sh

```diff
@@ -216,6 +216,11 @@ spec:
 EOF
   # Run only Windows tests
   ginko_params="$ginko_params --ginkgo.focus=Windows"
+
+elif [[ $TARGET =~ multus.* ]] || [[ $TARGET =~ genie.* ]]; then
+  # Run networking tests only (general networking, multus, genie, ...)
+  # If multus or genie is not present the test will  be skipped base on per-test checks
+  ginko_params="$ginko_params --ginkgo.focus=Networking"
```

Comment:
maybe also add the "Expose" tests? make sure that services play nicely with multus and genie?
```
--ginkgo.focus=Networking|Expose"
```

Reply:
Fixed, thx.